packages/@okta/vuepress-site/guides/custom-hosted-signin/customization-examples/index.md

```diff
@@ -0,0 +1,96 @@
+    {{{SignInWidgetResources}}}
+
+    <!-- Custom styles for your sign-in page -->
+    <link rel="stylesheet" type="text/css" href="http://example.com/yourCustomStyleSheet.css">
+</head>
+...
+```
+
+## Add Your Own Scripts
+You can add custom JavaScript code to your sign-in page. In the embedded HTML editor, add a script in the `<head>` section, below the <span v-pre>`{{{SignInWidgetResources}}}`</span> line.
+
+Example:
+```html
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="none" />
+    <title>{{pageTitle}}</title>
+    {{{SignInWidgetResources}}}
+
+    <!-- Add your scripts here -->
+    <script src="https://example.com/yourCustomScript.js"></script>
+    
+    <!-- Or, inline scripts -->
+    <script>
+      // Do something
+    </script>
+</head>
+...
+```
+
+## Per-Application Customization
+The Okta-hosted sign-in page is application-aware. This means that your client-side customizations can understand which application caused the sign-in page to load. This is useful when you have multiple applications or brands that you want to support.
+
+When the page renders, an object called `OktaUtil` exists on the page. By calling the `OktaUtil.getRequestContext()` method, scripts on the page can get details about the current request.
+
+For an OpenID Connect application, the application's client ID is stored in the request context's target for that object:
+
+```
+OktaUtil.getRequestContext().target.clientId
+```
+
+There is also additional information loaded in the `target`, such as `label`.
+
+> Note: The `getRequestContext()` method only returns a value when the Okta-hosted sign-in page is loaded in the context of an application (such as SP-initiated flows in SAML or the `/authorize` route for OpenID Connect). Otherwise, it returns undefined.
+
+To access the application's client ID (which uniquely identifies the application), write a function to safely get `target.clientId` from the request context:
+
+```html
+<script>
+  function getClientId() {
+    if (!OktaUtil) return undefined;
+    
+    var requestContext = OktaUtil.getRequestContext();
+    if (requestContext && requestContext.target && requestContext.target.clientId) {
+      return requestContext.target.clientId;
+    }
+  }
+</script>
+```
+
+Elsewhere in your page, using the method above, you can inspect the client ID and take action. For example, if you had a CSS file on your server that was for a particular client's CSS:
+
+```html
+<script>
+  var clientId = getClientId();
+
+  if (clientId === '00exampleclientid'){
+    // add application-specific CSS
+    var head = document.head;
+    var link = document.createElement('link');
+
+    link.type = 'text/css';
+    link.rel = 'stylesheet';
+    link.href = 'https://example.com/styles/' + client_id + '.css';
+    head.appendChild(link);
+  }
+</script>
+```
+
+<NextSectionLink/>
```

Comment:
This button renders as "Next: Next Steps". Can you use the pattern in other guides of explicitly labeling the button "Next Steps"?